ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -1566,6 +1566,84 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/release-type: informing
+    job-release: "4.3"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-ocp-installer-aws-fips-unit-4.3
```

Comment:
this job is a guess :woman_shrugging:

Reply:
job did not execute, got `2019/10/29 12:47:36 step <*steps.templateExecutionStep> is missing dependencies: <&api.internalImageLink{image:"src"}>`  so not sure I can configure the src template here?